### PR TITLE
Update to latest 1.10.3 stable version

### PR DESCRIPTION
**Pull request by dharamsk**
Many bug fixes and improvements included.

### CODE REVIEW

File: airflow/contrib/hooks/bigquery_hook.py [hunk 1]
```diff
@@ -60,6 +60,7 @@ def __init__(self,
             gcp_conn_id=bigquery_conn_id, delegate_to=delegate_to)
```
Comment (dharamsk) [inline airflow/contrib/hooks/bigquery_hook.py:60]: .

File: airflow/contrib/hooks/gcs_hook.py [hunk 2]
```diff
@@ -17,26 +17,25 @@
 # specific language governing permissions and limitations
```
Comment (dharamsk) [inline airflow/contrib/hooks/gcs_hook.py:17]: this is the piece that should fix the issues we're seeing with etl postal bq

File: airflow/contrib/kubernetes/kubernetes_request_factory/kubernetes_request_factory.py [hunk 3]
```diff
@@ -58,13 +58,12 @@ def add_secret_to_env(env, secret):
         })
```
Comment (dharamsk) [inline airflow/contrib/kubernetes/kubernetes_request_factory/kubernetes_request_factory.py:58]: .

File: airflow/contrib/operators/bigquery_operator.py [hunk 4]
```diff
@@ -16,15 +16,35 @@
 # KIND, either express or implied.  See the License for the
```
Comment (dharamsk) [inline airflow/contrib/operators/bigquery_operator.py:16]: .

File: airflow/contrib/operators/postgres_to_gcs_operator.py [hunk 5]
```diff
@@ -16,241 +16,89 @@
 # KIND, either express or implied.  See the License for the
```
Comment (dharamsk) [inline airflow/contrib/operators/postgres_to_gcs_operator.py:16]: jesus, does this work?

File: airflow/operators/latest_only_operator.py [hunk 6]
```diff
@@ -40,7 +40,7 @@ def execute(self, context):
             self.log.info("Externally triggered DAG_Run: allowing execution to proceed.")
             return
 
-        now = timezone.utcnow()
+        now = pendulum.utcnow()
```
Comment (dharamsk) [inline airflow/operators/latest_only_operator.py:43]: is this going to break shit?

File: airflow/operators/python_operator.py [hunk 7]
```diff
@@ -25,13 +25,12 @@
 import types
```
Comment (dharamsk) [inline airflow/operators/python_operator.py:25]: .

**Review by Wirick — APPROVED** [review]

**Pull request merged**